modules/order/config/install/views.view.commerce_orders.yml

```diff
@@ -788,6 +788,71 @@ display:
           destination: true
           entity_type: commerce_order
           plugin_id: entity_operations
+        name:
```

Comment:
@bojanz we'll only be able to search by contact mail without it. The combined filters filter requires fields in the "Fields" section.